### ISSUE TITLE
Does not work correctly with Ruby 1.9

### ISSUE THREAD

**Issue by donaldpiret** [texel/docusign#2]
Uses the older Soap4R Library which is helplessly broken on 1.9 (many fixed branched out there but no authoritative version).
Would be nice if a rewrite to Savon was possible

**Comment by texel** [comment]
Yeah, that sucks. Unfortunately, Savon doesn't generate proxy classes, which is more lightweight and "Ruby-like" but also makes it a lot harder to add extra functionality into the API wrapper. I have no need for an updated Docusign gem, so if you'd like to try and port it to 1.9 and another library like Savon, I'm sure others would appreciate it too.
